packages/core/src/interaction-output.ts

```diff
@@ -122,14 +124,14 @@ export class InteractionOutput implements WoT.InteractionOutput {
         // validate the schema
         const validate = ajv.compile<T>(this.schema);
 
-        if (!validate(json)) {
+        if (!this.ignoreValidation && !validate(json)) {
```

Comment:
I would skip schema compilation since in the case of `ingnoreValidation === true` is completely useless.

Reply:
Mhh, in the case of `ignoreValidation === true` the part after  `&&` is no longer taken into account since the `if` cannot turn into `true` anymore?

Or did I misunderstand the comment?